tox.ini

```diff
@@ -52,11 +52,14 @@ commands =
 
 # isort is an imports sorter for python: https://github.com/timothycrosley/isort
 # The following target sorts the import according to .isort.cfg file.
+# Pin isort to the last release prior to 5.0.0, which resulted in import
+# statements being flagged as improperly sorted where previously they were
+# not. See https://timothycrosley.github.io/isort/docs/major_releases/introducing_isort_5/.
 [testenv:isort]
 basepython = python3
 skip_install = true
 deps =
-    isort
+    isort==4.3.21
```

Comment:
Shall we apply changes introduced by isort 5 also in wip/hit branch? We might just commit those in a separate commit

Reply:
Sure, doing a separate commit